src/OpenTK/OpenTK.iOS.csproj

```diff
@@ -24,6 +26,7 @@
     <ConsolePause>false</ConsolePause>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DocumentationFile>bin\Debug\iOS\OpenTK.xml</DocumentationFile>
+    xed
```

Comment:
Oops. I was editing it with xed, which is a notepad++ equivalent on Linux. I usually do it in the terminal, but I must've had another window focused. Removing.